controller_manager/scripts/spawner

```diff
@@ -41,6 +41,7 @@ import yaml
 from controller_manager_msgs.srv import *
 from std_msgs.msg import *
 import argparse
+import controller_manager.controller_manager_interface
```

Comment:
please  use the same approach for listing controllers as used for the other services.

Reply:
(or do not call the service at all, see https://github.com/ros-controls/ros_control/pull/364#issuecomment-431430987)

Reply:
Done. See ee4cf92